fontbe/src/gvar.rs

```diff
@@ -0,0 +1,101 @@
+//! Generates a [gvar](https://learn.microsoft.com/en-us/typography/opentype/spec/gvar) table.
+
+use std::cmp::{min, max};
+
+use font_types::{F2Dot14, GlyphId};
+use fontdrasil::orchestration::Work;
+use fontir::coords::NormalizedLocation;
+use write_fonts::{
+    dump_table,
+    tables::{
+        gvar::{GlyphDeltas, GlyphVariations, Gvar},
+        variations::Tuple,
+    },
+};
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Bytes, Context, GvarFragment},
+};
+
+struct GvarWork {}
+
+pub fn create_gvar_work() -> Box<BeWork> {
+    Box::new(GvarWork {})
+}
+
+fn to_tuple(loc: &NormalizedLocation) -> Tuple {
+    Tuple::new(
+        loc.iter()
+            .map(|(_, nc)| F2Dot14::from_f32(nc.to_f32()))
+            .collect(),
+    )
+}
+
+fn to_deltas(fragment: &GvarFragment) -> Vec<GlyphDeltas> {
+    fragment
+        .deltas
+        .iter()
+        .filter_map(|(loc, deltas)| {
+            if !loc.has_any_non_zero() {
+                return None;
+            }
+
+            // TODO: add phantoms properly
```

Comment:
this is where fontTools computes the phantom points (needs hmtx)

https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/ttLib/tables/_g_l_y_f.py#L335-L367

Reply:
Copied the ref in, ty. I don't have the full capability yet.